libs/core/src/components/sage-avatar/stories/sage-avatar.stories.js

```diff
@@ -0,0 +1,48 @@
+import { html } from 'lit-html';
+import imgFile from '../assets/demi_wilkinson.jpg';
+
+const BaseTemplate = (args) => html`<sage-avatar
+	alt="${args.alt}"
+	badge="${args.badge}"
+	image="${args.image}"
+	size="${args.size}"
+	variant="${args.variant}"
+>
+</sage-avatar>`;
+
+const AdminTemplate = (args) => html`<sage-avatar size="${args.size}" variant="${args.variant}"></sage-avatar>`;
+
+const BadgeTemplate = (args) => html`<sage-avatar badge="${args.badge}" size="${args.size}"></sage-avatar>`;
+
+const CustomSizeTemplate = (args) => html`<sage-avatar style="--size: 128px;"></sage-avatar>`;
+
+const ImageTemplate = (args) => html`<sage-avatar alt="${args.alt}" image="${args.image}" size="${args.size}"></sage-avatar>`;
```

Comment:
I think with the templates defined this way it's messing with the storybook controls slightly. I am not positive, but for example, when looking at the image on the canvas tab, switching variants does not seem to apply. 

I think you could add additional props to the templates or update each to use the BaseTemplate. Let me know if that makes no sense.

Reply:
This should be fixed now.